service_taxi/src/main/kotlin/org/thechance/service_taxi/domain/usecase/TaxiManagementsUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package org.thechance.service_taxi.domain.usecase
+
+import org.thechance.service_taxi.domain.entity.Taxi
+import org.thechance.service_taxi.domain.entity.Trip
+
+interface TaxiManagementsUseCase {
+
+    //region Trip
+    suspend fun getClientTripsHistory(clientId: String, page: Int, limit: Int): List<Trip>
```

Comment:
shouldn't trips udecases be in `TripManagementsUseCase`

Reply:
just admin can show history i think, so should be stay in TaxiManagementsUseCase

Reply:
can admin also delete? `suspend fun deleteTrip(tripId: String): Boolean`